docs/RELEASING.md

```diff
@@ -83,6 +83,25 @@
    of the previous entry and adjusting `versionName`, `versionCode`, and `commit` (use the tag name).
    Leave `CurrentVersion` and `CurrentVersionCode` unchanged. Those specify which version is the stable/recommended
    build.
+
+   Example:
+
+   ```yaml
+   - versionName: "{versionName}"
+     versionCode: { versionCode }
+     commit: "{tagName}"
+     subdir: app-k9mail
+     prebuild: ( cd .. && ln -s app-metadata/com.fsck.k9 metadata )
```

Comment:
Note: If this doesn't work, we can add the symlink to the repository, so that we have a working short-term solution.